Client/nuxt.config.ts

```diff
@@ -10,11 +10,7 @@ export default defineNuxtConfig({
   },
   runtimeConfig: {
     public: {
-      baseURL: process.env.BASE_URL || 'https://localhost:7182',
-    }
+      baseURL: process.env.BASE_URL || 'https://localhost:5001',
+    },
   },
-  dirs: [
```

Comment:
Waarom is dit verwijderd? Dit zorgt ervoor dat alle composables zonder import in een component gebruikt kan worden.